apps/blog/package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "@litespace/blog",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev --turbo",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint"
+  },
+  "dependencies": {
+    "@hookform/resolvers": "^3.4.2",
+    "@radix-ui/react-dialog": "^1.0.5",
+    "@radix-ui/react-dropdown-menu": "^2.0.6",
+    "@radix-ui/react-label": "^2.0.2",
+    "@radix-ui/react-navigation-menu": "^1.1.4",
+    "@radix-ui/react-scroll-area": "^1.0.5",
+    "@radix-ui/react-select": "^2.0.0",
+    "@radix-ui/react-separator": "^1.0.3",
+    "@radix-ui/react-slot": "^1.0.2",
+    "@vercel/analytics": "^1.3.1",
+    "class-variance-authority": "^0.7.0",
+    "clsx": "^2.1.1",
+    "lucide-react": "^0.344.0",
+    "next": "^14.2.12",
+    "next-themes": "^0.3.0",
+    "query-string": "^9.0.0",
+    "react": "^18.3.1",
```

Comment:
Make sure that `react` has the same version like all of other projects located at `apps/*`. Also make sure that `next` version in `apps/blog` is the same as in `apps/neubla`